src/library/librarycontrol.cpp

```diff
@@ -325,6 +327,23 @@ void LibraryControl::bindLibraryWidget(WLibrary* pLibraryWidget, KeyboardEventFi
             &LibraryControl::libraryWidgetDeleted);
 }
 
+void LibraryControl::bindSearchboxWidget(WSearchLineEdit* pSearchbox) {
+    if (m_pSearchbox) {
+        disconnect(m_pSearchbox, 0, this, 0);
+    }
+    m_pSearchbox = pSearchbox;
+    connect(this,
+            &LibraryControl::clearSearch,
+            m_pSearchbox,
+            &WSearchLineEdit::clearSearch);
+    connect(m_pSearchbox,
+            &WSearchLineEdit::destroyed,
+            this,
+            &LibraryControl::libraryWidgetDeleted);
```

Comment:
wrong!
will fix it..
just discovered a recurring crasher, dunno if it's related.